### PR TITLE
Plugin to set font color

### PR DESCRIPTION
**Pull request by bampakoa**
It uses [angular-bootstrap-colorpicker](https://github.com/buberdds/angular-bootstrap-colorpicker) library.

screenshot from 2015-09-28 21 46 05

### CODE REVIEW

**Comment by bampakoa** [comment]
@bolelamx you were right about the **li** tag! It was left from a previous version of the library.

**Comment by stevermeister** [comment]
@bampakoa 
could I also ask you to remove dependancy for `angular-bootstrap-colorpicker` and added separate demo file for your colorpicker plugin where included colorpicker library from CDN

**Comment by bampakoa** [comment]
@stevermeister By separate demo file, you mean separate **index.html** in the src directory or **demo.html** in the dist one?

**Comment by stevermeister** [comment]
yes, I think **demo-plugin-colorpicker.html** in dist folder would be the best place

**Comment by bampakoa** [comment]
@stevermeister I removed the dependencies and added the demo file. I did not execute the **build** task, in order not to break existing dist files.

File: src/index.html [hunk 1]
```diff
@@ -40,5 +40,6 @@
 <script src="javascript/app/ng-wig/ngWigToolbarProvider.js"></script>
 <script src="javascript/app/ng-wig/ngWigPluginDirective.js"></script>
 <script src="javascript/app/plugins/formats.ngWig.js"></script>
+<script src="javascript/app/plugins/forecolor.ngWig.js"></script>
```
Comment (stevermeister) [inline src/index.html]: I thought you would include it in demo-plugin-colorpicker.html to show how easy is to work: switch off and on some of them. 

I think "formats" will also go to own demo file. It was just first plugin

**Comment by stevermeister** [comment]
@bampakoa,

thank you for your addition! 
it will be added in the next release

**Pull request merged**